### ISSUE TITLE
Default starter not upgrading to latest Gatsby packages

### ISSUE THREAD

**Issue by MWalid** [gatsbyjs/gatsby#17591]
I disconnected my internet, refreshed https://www.gatsbyjs.org and got site cannot be reached!

Even tried with `gatsby-default-starter`, enabled `gatsby-plugin-offline` on https site, and still, the site is not served when offline!  https://thetechchannels.com/bc/

I was expecting 100% offline support, something like: https://deanhume.com/create-a-really-really-simple-offline-page-using-service-workers/, but maybe I'm not getting the idea behind `gatsby-plugin-offline`.

## Summary
`gatsby-plugin-offline` does not serve sites when offline.

### File contents (if changed)

`gatsby-config.js`: 
```
module.exports = {
  siteMetadata: {
    title: `Gatsby Default Starter`,
    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
    author: `@gatsbyjs`,
  },
  plugins: [
    `gatsby-plugin-react-helmet`,
    {
      resolve: `gatsby-source-filesystem`,
      options: {
        name: `images`,
        path: `${__dirname}/src/images`,
      },
    },
    `gatsby-transformer-sharp`,
    `gatsby-plugin-sharp`,
    {
      resolve: `gatsby-plugin-manifest`,
      options: {
        name: `gatsby-starter-default`,
        short_name: `starter`,
        start_url: `/`,
        background_color: `#663399`,
        theme_color: `#663399`,
        display: `minimal-ui`,
        icon: `src/images/gatsby-icon.png`, // This path is relative to the root of the site.
      },
    },
    // this (optional) plugin enables Progressive Web App + Offline functionality
    // To learn more, visit: https://gatsby.dev/offline
    `gatsby-plugin-offline`,
  ],
}

```
`package.json`: N/A 
`gatsby-node.js`: N/A 
`gatsby-browser.js`: N/A 
`gatsby-ssr.js`: N/A

**Comment by davidbailey00** [comment]
Hey there, thanks for opening this!

The issue you've reported was actually fixed in #17408 - however, there seems to be an issue on our side that the default starter is still using v2 of the offline plugin, rather than v3 which has been released with this fix.

For now just run `yarn upgrade --latest` or `yarn install gatsby-plugin-offline@latest` inside your repository, which should fix the issue.

**Comment by MWalid** [comment]
Thanks for your response, great news its been resolved!

**Comment by davidbailey00** [comment]
Closed in #16840
